core/src/main/proto/Metric.proto

```diff
@@ -0,0 +1,17 @@
+﻿syntax = "proto3";
+package contracts;
+
+option csharp_namespace = "Microsoft.LocalForwarder.Library.Inputs.Contracts";
+option java_package = "com.microsoft.localforwarder.library.inputs.contracts";
+option java_multiple_files = true;
+
+import "DataPoint.proto";
+
+message Metric {
+	int32 ver = 1;
+    
+	// only one metric in the list is currently supported by Application Insights storage. If multiple data points were sent only the first one will be used.
+	repeated DataPoint metrics = 2;

```

Comment:
I still believe that this List<DataPoint> should be depreciated from the MetricData class which is creating this non-sense. Indeed we can bring in all the fields in the MetricData class inside the DataPoint class and just deprecate the whole class itself. It is of no use IMO and is just taking Memory footprint. Anyways, can be a conversation for seperate PR.